src/frontend/web_application/src/scenes/ApplicationSettings/components/ContactSettings/presenter.jsx

```diff
@@ -44,22 +44,22 @@ class ContactsForm extends Component {
     return (
       <FormGrid className="m-contacts-form">
         <FormRow>
-          <FormColumn size="shrink" bottomSpace >
+          <FormColumn rightSpace={false} bottomSpace >
```

Comment:
I don't like much the `rightSpace` modifier

no idea how to do it the "right way" for now